practice2.py

```diff
@@ -0,0 +1,9 @@
+def mysum(*num):

```

Comment:
Can you explain the use case of `(*num)` here?

Reply:
> 
> 
> Can you explain the use case of `(*num)` here?

*num allows you to do is take in more arguments than the number of formal arguments. With *num, any number of extra arguments can be tacked on to your current formal parameters.

Reply:
Thanks for the update! ✅